components/array-object-table/src/stories.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs/react';
+import { WithDocsCustom } from '@govuk-react/storybook-components';
+
+import ArrayObjectTable from '.';
+import ReadMe from '../README.md';
+
+const stories = storiesOf('Tables/ArrayObjectTable', module);
+const examples = storiesOf('Tables/ArrayObjectTable/Examples', module);
+
+stories.addDecorator(WithDocsCustom(ReadMe));
+stories.addDecorator(withKnobs);
+
```

Comment:
two lines of whitespace should be against our linting rules, i believe

Reply:
(i'm not against using two lines like this to help enforce a division, but rules iz rulez)